docs/plugins/testing_and_publishing/deploy.md

```diff
@@ -26,15 +25,11 @@ development process, see the [napari hub GitHub’s
 Wiki](https://github.com/chanzuckerberg/napari-hub/wiki).
 
 If you want your plugin to be available on PyPI, but not visible on the napari
-hub, you can add a `.napari/config.yml` file to the root of your repository with
-a visibility key. For details, see the [customization
+hub, add a `visibility` flag to your plugin manifest. For more details, see the 
```

Comment:
should link to manifest reference

Reply:
I think you're already covered:
https://github.com/napari/docs/blob/e9e874fdd88ab9cb7bc4b12d894cd4b6749b059c/docs/plugins/technical_references/manifest.md?plain=1#L1-L4

so `plugin-manifest` should work.

Reply:
In fact, it's used already here:
https://napari.org/dev/plugins/technical_references/index.html
and points to:
https://napari.org/dev/plugins/technical_references/manifest.html#plugin-manifest